app/actors/hyrax/actors/base_actor.rb

```diff
@@ -16,9 +16,16 @@ class BaseActor < AbstractActor
       # @param [Hyrax::Actors::Environment] env
       # @return [Boolean] true if create was successful
       def create(env)
-        apply_creation_data_to_curation_concern(env)
-        apply_save_data_to_curation_concern(env)
-        save(env) && next_actor.create(env) && run_callbacks(:after_create_concern, env)
+        begin
+          change_set = Hyrax::DynamicChangeSet.new(env.curation_concern)
```

Comment:
This seems like unnecessary indirection.  Can we make it `ChangeSetFactory.create(env.curation_concern)` instead?

Reply:
Ugh, I see this is how figgy is doing it, so I'm not going to complain too hard.

Reply:
I'm confused by this since it appears to be returning `Hyrax::DynamicChangeSet#initialize` with `env.curation_concern` being set as the `resource_param` when I'm guessing you're wanting the result of `Hyrax::DynamicChangeSet#new` and to get that right now you'd need to do `change_set = Hyrax::DynamicChangeSet.new.new(env.curation_concern)`.

Reply:
@carrickr I see the new changes :+1:

Reply:
We updated the DynamicChangeSet to use new as a class method instead of an instance method and avoid the `.new.new`

https://github.com/samvera/hyrax/blob/1729-changesets/app/change_sets/hyrax/dynamic_change_set.rb#L5

Reply:
@jcoyne You're welcome to use `ChangeSetFactory`. It's a `DynamicChangeSet` in Figgy because I tend to treat classes as factories.

@cjcolvar `DynamicChangeSet` overrides the class method `new`